smoketest/scripts/cli/test_protocols_segment-routing.py

```diff
@@ -47,14 +48,64 @@ def tearDown(self):
     def test_srv6(self):
         interfaces = Section.interfaces('ethernet', vlan=False)
         locators = {
-            'foo' : { 'prefix' : '2001:a::/64' },
-            'foo' : { 'prefix' : '2001:b::/64', 'usid' : {} },
+            'foo': {'prefix': '2001:a::/64'},
```

Comment:
Can we start with all numbers? foo1 - foo4?

Reply:
Sure, I was following the existing pattern.